eventActions/afkMessageCheckAction.js

```diff
@@ -41,16 +41,33 @@ class afkMessageCheckAction {
 			.setColor('#FFEC09');
 		const user = message.author;
 
+		// Returns a single integer rounded down for the difference in minutes between two Date.now() timestamps
+		function timedifference(timestamp1, timestamp2) {
+			var timestamp1 = new Date(parseInt(timestamp1));
+			var timestamp2 = new Date(parseInt(timestamp2));
+
+			var difference = timestamp2.getTime() - timestamp1.getTime();
+
+			var difference = Math.floor(difference/1000/60);
```

Comment:
same here with difference, but make it

`let difference = timestamp2 - timestamp1`
`difference = math.floor`